model-optimizer/extensions/middle/PoolV2ToAttributedPool.py

```diff
@@ -0,0 +1,56 @@
+"""
+ Copyright (C) 2018-2021 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from extensions.middle.PartialInfer import PartialInfer
+from mo.graph.graph import Graph
+from mo.graph.graph import rename_nodes
+from mo.middle.replacement import MiddleReplacementPattern
+from mo.ops.pooling import Pooling
+
+
+class PoolV2ToAttributedPool(MiddleReplacementPattern):
+    enabled = True
+
+    def run_after(self):
+        return [PartialInfer]
+
+    def run_before(self):
+        from extensions.middle.ApplyPermutations import ApplyPermutation
+        return [ApplyPermutation]
```

Comment:
None of these dependencies is needed. By default the Middle transformation is executed after `PartialInfer` and before `ApplyPermutation`

Reply:
Removed